sql/src/main/java/org/apache/druid/sql/guice/SqlModule.java

```diff
@@ -130,4 +137,54 @@ private boolean isAvaticaEnabled()
     Preconditions.checkNotNull(props, "props");
     return Boolean.valueOf(props.getProperty(PROPERTY_SQL_ENABLE_AVATICA, "true"));
   }
+
+  /**
+   * We create a new class for this module so that it can be shared by tests.  The structuring of the SqlModule
+   * at time of writing was not conducive to reuse in test code, so, instead of fixing that we just take the easy
+   * way out of adding the test-reusable code to this module and reuse that.
+   */
+  public static class SqlStatementFactoryModule implements Module
```

Comment:
Thanks for this. This is some fancy Guice wrangling that makes the setup a bit cleaner.